src/git.zsh

```diff
@@ -79,97 +82,96 @@ _fzf_complete_git() {
         return
     fi
 
-    if [[ ${(Q)${(z)arguments}} =~ '^git (branch|cherry-pick|merge|revert)' ]]; then
+    if [[ $subcommand =~ '(branch|cherry-pick|merge|revert)' ]]; then
         _fzf_complete_git-commits '--multi' $@
         return
     fi
 
-    if [[ ${(Q)${(z)arguments}} = 'git commit'* ]]; then
+    if [[ $subcommand = 'commit' ]]; then
+        local result
         if [[ ${${(Q)${(z)arguments}}[(r)--]} = -- ]]; then
             _fzf_complete_git-unstaged-files '--untracked-files=no' "--multi $_fzf_complete_preview_git_diff $FZF_DEFAULT_OPTS" $@
             return
         fi
 
-        if [[ $prefix =~ '^--(fixup|reedit-message|reuse-message|squash)=' ]]; then
-            prefix_option=${prefix/=*/=} _fzf_complete_git-commits '' $@
+        local using_commit=(c C fixup reedit-message reuse-message squash)
```

Comment:
The naming `using_*` is ambiguous. How about naming like:
- `git_options_commit_completion`
- `git_options_commit_message_completion`
- `git_options_none_completion`
- `git_options_path_completion`
- ...